diff --git a/node_modules/tldraw/dist-cjs/lib/shapes/arrow/ArrowShapeUtil.js b/node_modules/tldraw/dist-cjs/lib/shapes/arrow/ArrowShapeUtil.js
index 0297300..16ab6fe 100644
--- a/node_modules/tldraw/dist-cjs/lib/shapes/arrow/ArrowShapeUtil.js
+++ b/node_modules/tldraw/dist-cjs/lib/shapes/arrow/ArrowShapeUtil.js
@@ -824,8 +824,8 @@ class ArrowShapeUtil extends import_editor.ShapeUtil {
           verticalAlign: "middle",
           labelColor: (0, import_editor.getColorValue)(theme, shape.props.labelColor, "solid"),
           richText: shape.props.richText,
-          bounds: (0, import_arrowLabel.getArrowLabelPosition)(this.editor, shape).box.clone().expandBy(-import_default_shape_constants.ARROW_LABEL_PADDING * shape.props.scale),
-          padding: 0,
+          bounds: (0, import_arrowLabel.getArrowLabelPosition)(this.editor, shape).box.clone(),
+          padding: import_default_shape_constants.ARROW_LABEL_PADDING * shape.props.scale,
           showTextOutline: this.options.showTextOutline
         }
       )
diff --git a/node_modules/tldraw/dist-cjs/lib/shapes/shared/RichTextLabel.js b/node_modules/tldraw/dist-cjs/lib/shapes/shared/RichTextLabel.js
index 36ccf40..2657fde 100644
--- a/node_modules/tldraw/dist-cjs/lib/shapes/shared/RichTextLabel.js
+++ b/node_modules/tldraw/dist-cjs/lib/shapes/shared/RichTextLabel.js
@@ -208,6 +208,7 @@ function RichTextSVG({
       y: bounds.minY,
       width: bounds.w,
       height: bounds.h,
+      overflow: "visible",
       className: (0, import_classnames.default)(
         "tl-export-embed-styles tl-rich-text tl-rich-text-svg",
         showTextOutline ? "tl-text__outline" : "tl-text__no-outline"
diff --git a/node_modules/tldraw/dist-esm/lib/shapes/arrow/ArrowShapeUtil.mjs b/node_modules/tldraw/dist-esm/lib/shapes/arrow/ArrowShapeUtil.mjs
index 1319030..e339e2c 100644
--- a/node_modules/tldraw/dist-esm/lib/shapes/arrow/ArrowShapeUtil.mjs
+++ b/node_modules/tldraw/dist-esm/lib/shapes/arrow/ArrowShapeUtil.mjs
@@ -842,8 +842,8 @@ class ArrowShapeUtil extends ShapeUtil {
           verticalAlign: "middle",
           labelColor: getColorValue(theme, shape.props.labelColor, "solid"),
           richText: shape.props.richText,
-          bounds: getArrowLabelPosition(this.editor, shape).box.clone().expandBy(-ARROW_LABEL_PADDING * shape.props.scale),
-          padding: 0,
+          bounds: getArrowLabelPosition(this.editor, shape).box.clone(),
+          padding: ARROW_LABEL_PADDING * shape.props.scale,
           showTextOutline: this.options.showTextOutline
         }
       )
diff --git a/node_modules/tldraw/dist-esm/lib/shapes/shared/RichTextLabel.mjs b/node_modules/tldraw/dist-esm/lib/shapes/shared/RichTextLabel.mjs
index 7c6e019..7c6515b 100644
--- a/node_modules/tldraw/dist-esm/lib/shapes/shared/RichTextLabel.mjs
+++ b/node_modules/tldraw/dist-esm/lib/shapes/shared/RichTextLabel.mjs
@@ -181,6 +181,7 @@ function RichTextSVG({
       y: bounds.minY,
       width: bounds.w,
       height: bounds.h,
+      overflow: "visible",
       className: classNames(
         "tl-export-embed-styles tl-rich-text tl-rich-text-svg",
         showTextOutline ? "tl-text__outline" : "tl-text__no-outline"
diff --git a/node_modules/tldraw/src/lib/shapes/arrow/ArrowShapeUtil.tsx b/node_modules/tldraw/src/lib/shapes/arrow/ArrowShapeUtil.tsx
index f4bf141..d80e9a2 100644
--- a/node_modules/tldraw/src/lib/shapes/arrow/ArrowShapeUtil.tsx
+++ b/node_modules/tldraw/src/lib/shapes/arrow/ArrowShapeUtil.tsx
@@ -1072,9 +1072,8 @@ export class ArrowShapeUtil extends ShapeUtil<TLArrowShape> {
 					labelColor={getColorValue(theme, shape.props.labelColor, 'solid')}
 					richText={shape.props.richText}
 					bounds={getArrowLabelPosition(this.editor, shape)
-						.box.clone()
-						.expandBy(-ARROW_LABEL_PADDING * shape.props.scale)}
-					padding={0}
+						.box.clone()}
+					padding={ARROW_LABEL_PADDING * shape.props.scale}
 					showTextOutline={this.options.showTextOutline}
 				/>
 			</g>
diff --git a/node_modules/tldraw/src/lib/shapes/shared/RichTextLabel.tsx b/node_modules/tldraw/src/lib/shapes/shared/RichTextLabel.tsx
index 48f58bb..19e7b52 100644
--- a/node_modules/tldraw/src/lib/shapes/shared/RichTextLabel.tsx
+++ b/node_modules/tldraw/src/lib/shapes/shared/RichTextLabel.tsx
@@ -266,6 +266,7 @@ export function RichTextSVG({
 			y={bounds.minY}
 			width={bounds.w}
 			height={bounds.h}
+			overflow="visible"
 			className={classNames(
 				'tl-export-embed-styles tl-rich-text tl-rich-text-svg',
 				showTextOutline ? 'tl-text__outline' : 'tl-text__no-outline'
